src/Mapbender/CoreBundle/Resources/public/mapbender.element.layertree.tree.js

```diff
@@ -956,10 +956,28 @@
                         lastItem.after(chkbox);
                         var inpchkbox = chkbox.find('.checkbox');
                         inpchkbox.data('dimension', item);
+                        initCheckbox.call(inpchkbox);
```

Comment:
Where is this global function from? Can't we require or import this in a "normal" way?

We use this global function also in:
* application-edit.js
* mapbender.element.wmceditor.js

Reply:
This function is defined in FOM/CoreBundle/Resources/public/js/widgets/checkbox.js in the global scope. In my opinion, we should move this to some kind of utility object or something similar. Defining something in the global scope without a namespace is always bad.

Reply:
I created an issue regarding this global ui helper functions https://github.com/mapbender/mapbender/issues/615